test/connectivity/modules/NetworkController/Program.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class Program
+    {
+        static readonly ILogger Log = Logger.Factory.CreateLogger<Program>();
```

Comment:
It feels like we can simplify the implementation a bit by having one interface for all Offline operations with the following 3 methods - 
- SetOffline
- SetOnline
- GetStatus

This can replace INetworkInterfaceCommand, IFirewallCommand and IController. Note - we will still have separate implementations for linux/win and network interface/firewall, etc (total 6), but all implement the same interface.

We don't have to do it now.. but something to think about.